docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -211,6 +216,15 @@ apiml.security.auth.provider: zosmf
 apiml.security.auth.zosmfServiceId: zosmf  # Replace me with the correct z/OSMF service id
 ```
 
+##### SAF Authentication Provider
+
+The `SAF Authentication Provider` allows API Gateway to authenticate directly with the z/OS SAF provider installed on the system. The user needs SAF account in order to authenticate.
```

Comment:
SAF is present on every z/OS system, so the comment for its installation is meaningless. there is no such thing like z/OS user without SAF user account.